src/content/whitepaper/index.md

```diff
@@ -12,11 +12,11 @@ _This introductory paper was originally published in 2014 by Vitalik Buterin, th
 
 _While several years old, we maintain this paper because it continues to serve as a useful reference and an accurate representation of Ethereum and its vision. To learn about the latest developments of Ethereum, and how changes to the protocol are made, we recommend [this guide](/learn/)._
 
-[Open the Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
+[Open the original Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
```

Comment:
```suggestion
[Open the Ethereum Whitepaper as a PDF](./whitepaper-pdf/Ethereum_White_Paper_-_Buterin_2014.pdf)
```

I don't agree with this change. What we had proposed in the original issue about reverting the whitepaper was meant to create one canonical whitepaper, every other version we don't consider to be the valid 'true' whitepaper. Also, this implies that the whitepaper that follows is somehow different, which it isn't (or at least shouldn't be)

Reply:
Makes sense to me. Thanks for the explanation.